### PR TITLE
[SPELLCHECKERS] implemented Leuven University Dutch Spellchecker

### PR DESCRIPTION
**Pull request by jerome-poisson**
text checking + suggestions have been implemented.

SDBELGA-122

### CODE REVIEW

File: superdesk/text_checkers/spellcheckers/leuven_dutch.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        r = requests.post(check_url, data=data)
+        if r.status_code != 200:
+            raise SuperdeskApiError.internalError("Unexpected return code from {}".format(self.name))
+
+        data = r.json()
+
+        err_list = []
+        check_data = {'errors': err_list}
+        for mistake in data['spellingchecker']['output'].get('mistakes', []):
+            idx = text.find(mistake)
```
Comment (petrjasek) [inline superdesk/text_checkers/spellcheckers/leuven_dutch.py]: what about case when you have same mistake multiple times in text?

Comment (jerome-poisson) [reply superdesk/text_checkers/spellcheckers/leuven_dutch.py]: only the first one will be underlined. I've actually thought about this one too, and I was thinking it was an acceptable compromise as it's an edge case and their output format is not that nice (marking and mistakes are not directly linked).

But after thinking about it again it should not be such a problem to parse and associate to spelling mistake, and it's more clean indeed, so I'll update the PR to work with the marked text.

Comment (jerome-poisson) [reply superdesk/text_checkers/spellcheckers/leuven_dutch.py]: @petrjasek I've changed the code it's not using `mistakes` anymore, only the marker. Now the same mistake done several times is underlined every time as expected.

File: superdesk/text_checkers/spellcheckers/leuven_dutch.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            raise SuperdeskApiError.internalError("Unexpected return code from {}".format(self.name))
+
+        data = r.json()
+
+        err_list = []
+        check_data = {'errors': err_list}
+        len_end_marker = len(END_MARKER)
+        output = data['spellingchecker']['output']
+        marked = output['marked'].split(START_MARKER)
+        # the first item in marded is unmarked text, we start our index there
```
Comment (petrjasek) [inline superdesk/text_checkers/spellcheckers/leuven_dutch.py]: typo `s/marded/marked/`

Comment (jerome-poisson) [reply superdesk/text_checkers/spellcheckers/leuven_dutch.py]: fixed, thanks

**Pull request merged**